cdp/wallet_address.py

```diff
@@ -72,6 +72,25 @@ def can_sign(self) -> bool:
         """
         return self.key is not None
 
+    def export(self) -> str:
+        """Export the wallet address's private key as a hex string.
+
+        Returns:
+            str: The wallet address's private key as a hex string.
+
+        Raises:
+            ValueError: If the wallet address does not have a private key.
+
+        """
+        local_account = self.key
+
+        if local_account is None:
+            raise ValueError("Account is unavailable")
+
+        key_bytes = local_account.key
+
+        return key_bytes.hex()
```

Comment:
does this `0x` prefix the hex string? Let's make sure that this is consistent across SDK's.

Reply:
roger that! added a test for 0x